frontend/src/Style.elm

```diff
@@ -0,0 +1,72 @@
+module Style exposing
+    ( appearanceNone
+    , opacity
+    , outlineNone
+    , rounded
+    , roundedExtraLarge
+    , roundedLarge
+    , shadowLarge
+    , style
+    , styleList
+    )
+
+import Html
+import Html.Attributes exposing (class, classList)
+
+
+style : List String -> Html.Attribute msg
```

Comment:
Ideally we wouldn't expose the fact that all of these styles are just Tailwind class names. We could hide this by using a custom type:

```elm
type Style = Style String
```

In the future, I could see this expanding to something like

```elm
type ClassStyle = ClassStyle String | ClassStyle (List ClassStyle)

{{- 
  Currently, State.hover can only take one style, the recursive type would 
  allow us to do something like the following
}}

style [
  Background.color Color.red
  , Style.State.hover [
    Background.color Color.blue
  ]
]

type Style = ClassStyle | CssStyle

{{- 
  As we gradually move away from Tailwind, CssStyle would allow us to
  specify styling with a library like elm-css.
 -}}

What's unclear is if `style` would return a single `Html.Attribute` or a list of them, ideally it would return a single attribute, to allow styles to be defined in line with the element definition without the need to concatenate attributes everywhere.
```